Dockerfile

```
@@ -10,9 +10,10 @@ COPY . /app
 FROM base as production
 ENV FLASK_ENV=production
 # Install the project dependencies
-RUN poetry install --no-dev
+RUN poetry config virtualenvs.create false --local && poetry install --no-dev
 # Run as production on startup of the container
-ENTRYPOINT ["poetry", "run", "gunicorn", "--config", "gunicorn_config.py", "todo_app.app:create_app()"]
+RUN chmod +x ./entrypoint.sh
+ENTRYPOINT ./entrypoint.sh
```

In this change you've added a required environment variable (`PORT`) that isn't documented anywhere, so your Readme instructions now fail. One nice solution to that might be specifying a default value in your Dockerfile; if the user provides one then it gets ignored but they can also get away without specifying it.
```suggestion
ENV PORT=5000
ENTRYPOINT ./entrypoint.sh
```